.github/workflows/ci-isotovideo.yml

```diff
@@ -10,3 +10,6 @@ jobs:
     - uses: actions/checkout@v2
     - name: Run isotovideo tests
       run: make test-isotovideo
+    - name: Error in isotovideo
+      if: ${{ failure() }}
+      run: echo 'isotovideo failed, something modified the default schedule or the output; run locally to find out more.' && exit 1
```

Comment:
Correct me if i am wrong but why the message cant be in the same line with `make test-isotovideo`?

Reply:
Nothing to correct, I guess the message can be in the same place, since the second run also executes both statements.